<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>windowAdd</class>
 <widget class="QDialog" name="windowAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>519</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>519</width>
    <height>450</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnAgregar">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLimpiar">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpiar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCerrar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitulo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Agregar nuevo</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>441</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Ingreso de datos</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>391</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QComboBox" name="cmbDispositivo"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblCorriente">
       <property name="text">
        <string>Corriente (A)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="txtVoltaje"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="txtPotencia"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="txtCorriente"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Voltaje (V)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblDispositivo">
       <property name="text">
        <string>Seleccionar dispositivo</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblTiempoUso">
       <property name="text">
        <string>Tiempo de uso (H/D)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="txtTiempoUso"/>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblPotencia">
       <property name="text">
        <string>Potencia (KW-H)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
